packages/client/hmi-client/src/components/model/petrinet/tera-initial-table-v2.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<!-- Stratified -->
+	<template v-if="isStratified">
+		<Accordion multiple>
+			<AccordionTab v-for="[key, values] in collapseInitials(props.mmt).entries()" :key="key">
+				<template #header>
+					<span>{{ key }}</span>
+					<Button label="Open Matrix" text size="small" @click.stop="openMatrix(key)" />
+				</template>
+				<div class="flex">
+					<Divider layout="vertical" type="solid" />
+					<ul>
+						<li v-for="initial in values" :key="initial">
+							<tera-initial-entry
+								:model-configuration="props.modelConfiguration"
+								:initial-id="initial"
+								@update-expression="emit('update-expression', $event)"
+								@update-source="emit('update-source', $event)"
+							/>
+							<Divider type="solid" />
+						</li>
+					</ul>
+				</div>
+			</AccordionTab>
+		</Accordion>
+	</template>
+
+	<!-- Unstratified -->
+	<template v-else>
```

Comment:
You can put the `v-if` and `v-else` in the html tags themselves since they contain the content, no need for the `<template>`